README.md

```diff
@@ -130,12 +146,15 @@ Therefore this section has been deliberately kept rather short. The libraries fe
 
 ### AFNetworking
 
-A perceived 99.95 percent of iOS developers use this network library. While `NSURLSession` is surprisingly powerful by itself, `AFNetworking` remains unbeaten when it comes to actually managing a queue of requests, which is pretty much a requirement in any modern app.
+A perceived 99.95 percent of iOS developers use this network library. While `NSURLSession` is surprisingly powerful by itself, [AFNetworking][afnetworking-github] remains unbeaten when it comes to actually managing a queue of requests, which is pretty much a requirement in any modern app.
```

Comment:
Should we mention about the async UIImageView helpers AFNetworking provides? Or is there an easier / better way nowadays?